Frameworks/Core/ERPrototypes/Resources/erprototypes.eomodeld/EOJDBCPostgresqlPrototypes.plist

```diff
@@ -359,6 +359,13 @@
             valueClassName = NSString; 
             valueType = S; 
             width = 10000000; 
+        }, 
+        {
+            columnName = uuid; 
+            externalType = bytea; 
```

Comment:
Paul,

PG UUID seems to map to Java UUID and is a very PG specific. I do not remember having tried to map a UUID column to an NSData with EOF.

Keep in mind that EOF only supports Integer, Long, String, Date and NSData as primary key type.

Samuel



> Le 28 juin 2016 à 07:19, Paul Hoadley <notifications@github.com> a écrit :
> 
> In Frameworks/Core/ERPrototypes/Resources/erprototypes.eomodeld/EOJDBCPostgresqlPrototypes.plist <https://github.com/wocommunity/wonder/pull/749#discussion_r68738897>:
> 
> > @@ -359,6 +359,13 @@
> >              valueClassName = NSString; 
> >              valueType = S; 
> >              width = 10000000; 
> > +        }, 
> > +        {
> > +            columnName = uuid; 
> > +            externalType = bytea; 
> Hi Samuel,
> 
> Would it be possible to use PostgreSQL's uuid <https://www.postgresql.org/docs/current/static/datatype-uuid.html> as the external type here? Or does it create problems elsewhere?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/wocommunity/wonder/pull/749/files/66093a103c504f7328258ea4c5a59052244c0d08..bde27aea1cebb6979543b5bedd9e0a78894dba38#r68738897>, or mute the thread <https://github.com/notifications/unsubscribe/AA-cjlxQd5Iz-penZKgBNvWmacj57584ks5qQQNXgaJpZM4In8EK>.
>